packages/relay/src/lib/clients/sdkClient.ts

```diff
@@ -515,6 +516,20 @@ export class SDKClient {
         throw predefined.HBAR_RATE_LIMIT_EXCEEDED;
       }
       throw sdkClientError;
+    } finally {
+      /**
+       *  For transactions of type CONTRACT_CREATE, if the contract's bytecode (calldata) exceeds 5120 bytes, HFS is employed to temporarily store the bytecode on the network.
+       *  After transaction execution, whether successful or not, any entity associated with the 'fileId' should be removed from the Hedera network.
+       */
+      if (transaction['_callDataFileId']) {
```

Comment:
Q: why not check for the transaction type and then just use [callDataFileId()](https://github.com/hashgraph/hedera-sdk-js/blob/develop/src/EthereumTransaction.js#L188) and check for a valid entityId
This would also ensure we catch if the sdk ever deprecates and breaks us

Reply:
I was looking around in the EthereumTransaction and Transaction extension classes but they don't seem like they carry transaction type around. 

However, @georgi-l95 suggested passing the `fileId` (generated by FileCreateTransaction()) down the path and this makes very good sense. We know for sure that if `fileId !== null` => there's a file to delete so we don't have to worry about the SDK every changing the property and breaks us later.